client/app/components/service-status/service-status-action.component.js

```diff
@@ -0,0 +1,15 @@
+(() => {
+  angular.module('directives').component('serviceStatusAction', {
+    templateUrl: 'components/service-status/service-status-action.component.html',
+    bindings: {
+      serviceInfos: '<',
+      hideRenewAction: '<',
+      forceHideRenewAction: '<',
+      hideRenewDate: '<',
+      serviceType: '<',
+      serviceName: '<',
+      inline: '<',
+    },
+    controller: 'serviceStatusActionComponentCtrl',
```

Comment:
You can attach the `Controller` to the `Component` without using the name if you want to :)
`import ctrl from './path/to/controller'`